### PR TITLE
Prepare 1.2.0 release

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY
Run add_docs.py
Run antsibull-changelog
Bump release in galaxy.yml
Bump required amazon.aws release in test requirements

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
community.aws

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -39,10 +57,10 @@ Minor Changes
 - aws_acm - Update automatic retries to stabilize the integration tests.
 - aws_codecommit - Support updating the description
 - aws_kms - Adds the ``enable_key_rotation`` option to enable or disable automatically key rotation.
-- aws_kms: code refactor, some error messages updated
+- aws_kms - code refactor, some error messages updated'
```
Comment (tremble) [inline CHANGELOG.rst:60]:
```suggestion
- aws_kms - code refactor, some error messages updated
```

**Review by tremble — CHANGES_REQUESTED** [review]
One minor thing, there's an extra ' lurking in one of the changelos entries, I suspect this could do something weird to the parser

File: CHANGELOG.rst (hunk 1 above)
Comment (jillr) [reply CHANGELOG.rst:60]: It's being generated by antsibull-changelog. I can manually remove it this release, and keep an eye if it pops back up again for 1.3.0 (or 1.2.1 if we have one).
changelogs/fragments/66037-aws_kms.yml in ansible/ansible was wrapped in single quotes so it's possible antsibull might not be expecting those?
cc @felixfontein

**Review by tremble — APPROVED** [review]
LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (felixfontein) [reply CHANGELOG.rst]: @jillr this was manually added in #185 by yourself: https://github.com/ansible-collections/community.aws/pull/185/commits/7cc15bc4f1934960cee8f363ec3305086dd4a0ce#diff-7c54aa032dfbb9dff42ef068f965d4d4L86-R80 (I guess you simply forgot to remove the trailing apostrophe.)

(antsibull-changelog is using pyyaml to load and store YAML files and does not modify the contents of changelog fragments)

Comment (jillr) [reply CHANGELOG.rst]: Oh no I missed that, thanks!  :)

**Pull request merged**